modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/snapshot/storage/LastGlobalSnapshotOrdinalStorage.scala

```diff
@@ -0,0 +1,30 @@
+package org.tessellation.dag.l1.domain.snapshot.storage
+
+import cats.Monad
+import cats.effect.Ref
+import cats.syntax.functor._
+
+import org.tessellation.dag.snapshot.SnapshotOrdinal
+
+trait LastGlobalSnapshotOrdinalStorage[F[_]] {
+  def set(snapshotOrdinal: SnapshotOrdinal): F[Unit]
+
+  def get: F[SnapshotOrdinal]
+}
+
+object LastGlobalSnapshotOrdinalStorage {
```

Comment:
Maybe in the future this will become a `GlobalSnapshotStorage`, but we will know after implementing download/redownload features.

Reply:
For pulling the global snapshot functionality that's all I need for now. Processing, verifying and storing global snapshot will be a subject of a different PR (download/redownload). I would rather decide then, how and what to store.

Reply:
So you download (pull) a global snapshot or just the ordinal? Name of the PR states you're fetching the global snapshot so I assume it implements (or should implement) download functionality